packages/ui/src/components/va-pagination/VaPagination.demo.vue

```diff
@@ -1,216 +1,376 @@
-                v-model="activePage"
+                direction-icon-left="volume_mute"
+                direction-icon-right="volume_down"
               />
               <va-pagination
+                v-model="activePage"
                 :pages="10"
                 :visible-pages="3"
-                boundary-icon-left='volume_off'
-                boundary-icon-right='volume_up'
-                v-model="activePage"
+                boundary-icon-left="volume_off"
+                boundary-icon-right="volume_up"
               />
               <va-pagination
+                v-model="activePage"
                 :pages="10"
                 :visible-pages="3"
-                boundary-icon-left='volume_off'
-                boundary-icon-right='volume_up'
-                direction-icon-right='volume_down'
-                direction-icon-left='volume_mute'
+                boundary-icon-left="volume_off"
+                boundary-icon-right="volume_up"
+                direction-icon-left="volume_mute"
+                direction-icon-right="volume_down"
+              />
+            </VbCard>
+          </td>
+        </tr>
+        <tr>
+          <td>Input</td>
+          <td>
+            <VbCard>
+              <va-pagination
                 v-model="activePage"
+                :pages="20"
+                input
               />
             </VbCard>
           </td>
         </tr>
-      </table>
-    </VbCard>
-    <VbCard
-      title="With input"
-    >
-      <va-pagination
-        :pages="20"
-        v-model="activePage"
-        input
-      />
-    </VbCard>
-    <VbCard
-      title="With boundary numbers"
-    >
-      <va-pagination
-        :pages="20"
-        :visible-pages="5"
-        v-model="activePage"
-        boundary-numbers
-      />
-      <va-pagination
-        :pages="20"
-        :visible-pages="7"
-        v-model="activePage"
-        boundary-numbers
-      />
-    </VbCard>
-    <VbCard
-      title="Flat"
-    >
-      <va-pagination
-        :pages="20"
-        :visible-pages="5"
-        v-model="activePage"
-        flat
-      />
-    </VbCard>
-    <VbCard
-      title="Flat with input"
-    >
-      <va-pagination
-        :pages="20"
-        v-model="activePage"
-        flat
-        :visible-pages="5"
-        input
-      />
-    </VbCard>
-    <VbCard
-      title="Hide on single page"
-    >
-      <va-pagination
-        :pages="1"
-        hide-on-single-page
-      />
-    </VbCard>
-    <VbCard
-      title="Stateful"
-    >
-      <va-pagination
-        :pages="10"
-        stateful
-        :visible-pages="5"
-      />
-    </VbCard>
-    <VbCard
-      title="Use context"
-    >
-      <va-config :components="{
+        <tr>
+          <td>Boundary Icons</td>
+          <td>
+            <VbCard>
+              <va-pagination
+                v-model="activePage"
+                :pages="20"
+                :visible-pages="5"
+                boundary-numbers
```

Comment:
<img width="653" alt="image" src="https://user-images.githubusercontent.com/5394573/189234348-1cb7b271-a069-4755-b689-42417ce8e795.png">

Only bottom demo moves boundary (`...`) on page selection. I assume both should move it.

Here also we might want an ability to remove navigation arrows, as they're somewhat redundant.

Reply:
It's seems to be old bug. Deal with it.